### DISCUSSION TITLE
To be aware of - could be added to docs - Primary constructors in Steps/Specs cause the VS Code plugin for Cucumber not to work

### DISCUSSION THREAD

**Discussion by dhansak79** [reqnroll/Reqnroll#313] [Contributor Q&A]
https://github.com/cucumber/vscode/issues/254

**Comment by gasparnagy** [comment]
The vscode extension parses C# with a C# parser lib that is probably not yet updated to latest C# magic...

Could you please send a quick PR to update the docs here?

**Comment by dhansak79** [comment]
@gasparnagy can you add me as a contributor? or would you prefer me to take a fork?

**Reply by gasparnagy** [reply]
Please do it from your own fork. According to our current guidelines, we add people to the contributors once they have an approved PR with code changes. See https://github.com/orgs/reqnroll/discussions/67.

**Comment by dhansak79** [comment]
I will raise an issue just now to attach to my PR.

**Reply by gasparnagy** [reply]
Super, thx! Although for simple doc updates it is not necessary to have an issue.
